response_operations_ui/templates/sign_in.html

```diff
@@ -4,72 +4,88 @@
-            <form method="post">
-                {{ form.csrf_token }}
-                <div>
-                    {% if form.errors %}
-                    <div class="panel panel--error">
-                        <div class="panel__header">
-                            {% if form.errors|length == 1 %}
-                            <h1 id="error-count" class="panel__title venus">There is 1 error</h1>
-                            {% else %}
-                            <h1 id="error-count" class="panel__title venus">There are {{ form.errors|length }} errors</h1>
-                            {% endif %}
+            <section>
+                <form method="post" class="form u-mt-m">
+                    {{ form.csrf_token }}
+                    <div>
+                        {% if form.errors %}
+                        <div class="panel panel--error u-mb-m">
+                            <div class="panel__header">
+                                {% if form.errors|length == 1 %}
+                                <h1 id="error-count" class="panel__title venus">There is 1 error on this page</h1>
+                                {% else %}
+                                <h1 id="error-count" class="panel__title venus">There are {{ form.errors|length }} errors on this page</h1>
+                                {% endif %}
+                            </div>
+                            <div class="panel__body">
+                                <ol>
+                                  {% for error in form.username.errors %}
+                                    <li class="panel__title venus"><a href='#username'>{{ error }}</a></li>
+                                  {% endfor %}
+                                  {% for error in form.password.errors %}
+                                    <li class="panel__title venus"><a href='#password'>{{ error }}</a></li>
+                                  {% endfor %}
+                                </ol>
+                            </div>
                         </div>
-                        <div class="panel__body">
-                            <ol>
-                              {% for error in form.username.errors %}
-                                <li class="panel__title venus"><a href='#username'>{{ error }}</a></li>
-                              {% endfor %}
-                              {% for error in form.password.errors %}
-                                <li class="panel__title venus"><a href='#password'>{{ error }}</a></li>
-                              {% endfor %}
-                            </ol>
+                        {% endif %}
+                        <h1 class="saturn">Sign in</h1>
+                        {% if messages %}
+                            {% for category, message in messages %}
+                                {% if category == "failed_authentication" %}
+                                <div id="username-panel" class="field panel panel--simple panel--error">
```

Comment:
Thsi is perfect candate for encapsulation , so the code will be more redable. You might have a function print_username_panel_error() where you actually do the same check 
 {% if messages %}
                            {% for category, message in messages %}
                                {% if category == "failed_authentication" %}
                                <div id="password-panel" class="field panel panel--simple panel--error u-mb-s">
                                {% endif %}
                            {% endfor %}
{% else %}